rust/generate_message/src/lib.rs

```diff
@@ -1065,14 +1065,16 @@
 //!
 //! `$ cargo run meta-at-block --url wss://westend-rpc.polkadot.io --block
 //! 780812df50c4006d1865742269fe4ca339c097e61d6279cce91ebc58f5aebada`
-#![deny(unused_crate_dependencies)]
+#![deny(unused)]
```

Comment:
There is a false-positive using `unused_crate_dependencies` https://github.com/rust-lang/rust/issues/72686